generated/java/google-cloud-speech-v1beta1/src/main/java/com/google/cloud/speech/spi/v1beta1/SpeechApi.java

```diff
@@ -146,6 +154,16 @@ public void close() throws IOException {
     }
   }
 
+  public final StreamObserver<StreamingRecognizeRequest> streamingRecognize(
+      StreamObserver<StreamingRecognizeResponse> streamObserver) {
+    return streamingRecognizeCallable.call(streamObserver);
+  }
+
+  public final ApiCallable<StreamObserver<StreamingRecognizeResponse>,
+      StreamObserver<StreamingRecognizeRequest>> streamingRecognizeCallable() {
+    return streamingRecognizeCallable;
+  }
```

Comment:
I think the parameter/return types of streamingRecognize() is a bit of confusing so only providing the callable might be better. I have no objections to it.